whoami/whoami.go

```diff
@@ -61,6 +61,17 @@ var (
 	//fatal = log.New(os.Stderr, "", log.Lshortfile)
 )
 
+func lookupUserName() string {
```

Comment:
Why do you want to put this into its own function?

If you run `go build -gcflags -m`, `lookupUserName` isn't inlined. Really, there's no measurable performance impact, but since `whoami` only does one thing, I don't see the point in having the logic in a separate function.